torch_audiomentations/core/composition.py

```diff
@@ -37,8 +39,13 @@ def forward(self, samples, sample_rate: typing.Optional[int] = None):
             if self.shuffle:
                 random.shuffle(transform_indexes)
             for i in transform_indexes:
-                samples = self.transforms[i](samples, sample_rate)
+                tfm = self.transforms[i]
+                if isinstance(tfm, BaseWaveformTransform):
+                    samples = self.transforms[i](samples, sample_rate)
+                else:
+                    samples = self.transforms[i](samples)
 
+                samples = self.transforms[i](samples, sample_rate)
```

Comment:
This line needs to be removed. The two tests that are currently failing will pass again after the removal

Reply:
Yes wow, should have seen that!